### PR TITLE
Removed user's show page

### PR DESCRIPTION
**Pull request by laszlokardinal-dina**

### CODE REVIEW

File: test/controllers/user_controller_test.exs [hunk 1]
```diff
@@ -113,32 +91,6 @@ defmodule CoursePlanner.UserControllerTest do
     assert html_response(conn, 403)
   end
 
-  test "does not update chosen user for non coordinator use", %{conn: _conn} do
```
Comment (ggpasqualino) [inline test/controllers/user_controller_test.exs]: I think this test could stay

File: web/controllers/user_controller.ex [hunk 2]
```diff
@@ -32,10 +23,11 @@ defmodule CoursePlanner.UserController do
 
     case Repo.update(changeset) do
       {:ok, user} ->
-        Users.notify_user(user, current_user, :user_modified, user_url(conn, :show, user))
+        user_url = user_show_url(user)
+        Users.notify_user(user, current_user, :user_modified, user_url)
```
Comment (ggpasqualino) [inline web/controllers/user_controller.ex]: the notification needs the full url, but `user_show_url` generates only the path

File: test/controllers/user_controller_test.exs (hunk 1 above)
Comment (laszlokardinal-dina) [inline test/controllers/user_controller_test.exs]: Sorry, that was a mistake. I put it back

File: web/controllers/user_controller.ex (hunk 2 above)
Comment (laszlokardinal-dina) [inline web/controllers/user_controller.ex]: I added CoursePlanner.Endpoint.url there

**Pull request merged**